testdata/project-v2/controllers/suite_test.go

```diff
@@ -22,14 +22,15 @@ import (
 
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
-
-	corev1 "k8s.io/api/core/v1"
 	"k8s.io/client-go/kubernetes/scheme"
 	"k8s.io/client-go/rest"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 	"sigs.k8s.io/controller-runtime/pkg/envtest"
 	logf "sigs.k8s.io/controller-runtime/pkg/log"
 	"sigs.k8s.io/controller-runtime/pkg/log/zap"
+
+	corev1 "k8s.io/api/core/v1"
```

Comment:
This is a workaround as I have been unable to make it join the previous group.

The reason is simple to understand, but not easy to solve. When `kubebuilder` scaffolds this file, all the imports are in a single group, and he is able to create the 3 separate groups (std/external/internal) correctly. But some imports are added aftwerwards through the `// +kubebuilder:scaffold:imports` tag. When they are added, some groups are already created, and `imports.Process` will not mix the groups together except for the standard library imports that are always placed at the top. So despite `corev1 "k8s.io/api/core/v1"` being a external import, it is placed in a separate group. I think that the easiest fix will be making file updates first remove every blank line in the imports section, thus eliminating the previously created groups, and letting `import.Process` recreate the groups.

Reply:
> The problem of this approach is that groups manually created by users will be removed.

That's fine, I think.

Do you want to do that here, or in a follow up?

Reply:
I would say in a follow up as that would require messing with templates manually.